tests/Unit/Discovery/OauthTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/******************************************************************************
+ * Copyright 2017 Okta, Inc.                                                  *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License");            *
+ * you may not use this file except in compliance with the License.           *
+ * You may obtain a copy of the License at                                    *
+ *                                                                            *
+ *      http://www.apache.org/licenses/LICENSE-2.0                            *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ ******************************************************************************/
+
+use Okta\JwtVerifier\Discovery\Oauth;
+
+class OauthTest extends BaseTestCase
+{
+    /** @test */
+    public function sets_well_known_correctly()
+    {
+        $oauth = new Oauth();
+
+        $this->assertEquals(
+            '/.well-known/oauth-authorization-server',
+            $oauth->getWellKnownUri(),
+            '.well-known endpoint is not set correctly'
```

Comment:
AHH, sorry. Right vs Left. Some of the test libs i use put the expected/actual on different sides, drives me crazy.

-php-noob